applications/nrf_desktop/src/modules/ble_latency.c

```diff
@@ -23,6 +23,7 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_LATENCY_LOG_LEVEL);
 #define LOW_LATENCY_CHECK_PERIOD_MS	K_SECONDS(5)
 #define DEFAULT_LATENCY			CONFIG_BT_PERIPHERAL_PREF_LATENCY
 #define DEFAULT_TIMEOUT			CONFIG_BT_PERIPHERAL_PREF_TIMEOUT
+/* 0x0d00 is a magic number which codes LLPM connection */
```

Comment:
do we need this comment? I do not think that it adds any more information than the define itself

Reply:
So you suggest somehow passing info that this is a magic number inside name - please suggest better naming then  REG_CONN_INTERVAL_LLPM_MASK. 

maybe LLPM_REG_CONN_INTERVAL_MAGIC_NUM?

Reply:
Also check Mares suggestion below.